packages/blade/src/components/Tooltip/utils.ts

```diff
@@ -6,4 +8,32 @@ const getPlacementParts = (placement: NonNullable<Placement>) => {
   return [side, alignment] as const;
 };
 
-export { getPlacementParts };
+/**
+ * Merges two sets of props.
+ *
+ * modified from: https://github.com/ariakit/ariakit/blob/f152585bbb698412ced42bcfa059038ef9d40100/packages/ariakit-react-core/src/utils/misc.ts#L48
+ */
+const mergeProps = <T extends Record<string, any>>(base: T, overrides: T) => {
```

Comment:
Some tests for this util might be helpful

Reply:
added